api/src/main/java/net/kyori/adventure/audience/EmptyAudience.java

```diff
@@ -51,6 +53,15 @@
     return defaultValue.get();
   }
 
+  @Override
+  public @NonNull Audience filterAudience(final @NonNull Predicate<? super Audience> filter) {
+    return this;
```

Comment:
Is there a reason why this method is overidden here? Even though it's empty, I'd still assume that it would be ran through the predicate.

Reply:
This audience has no identity of its own -- it's like running `forEach` on an empty list.

Reply:
I'm not quite sure it is, I think it's more like the equivalent of skipping nulls on a during a `forEach` on a list with nullable elements.